parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -57,4 +57,8 @@ cleanUpGPUDrivers() {
     rm -Rf $GPU_DEST
 }
 
+listInstalledPackages() {
```

Comment:
this is never used in CSE right? can we possibly move it to tools-install for VHD build instead?

Reply:
Good point, moved these functions into the tool_installs_distro scripts.